bftengine/src/bcstatetransfer/RVBManager.cpp

```diff
@@ -211,6 +213,9 @@ void RVBManager::pruneRvbDataDuringCheckpoint(const CheckpointDesc& new_checkpoi
     }
     LOG_DEBUG(logger_, oss.str());
   }
+  metrics_.pruning_vector_elements_count_.Get().Set(pruned_blocks_digests_.size());
+  metrics_.pruning_vector_size_in_bytes_.Get().Set(pruned_blocks_digests_.size() * (sizeof(BlockId) + sizeof(Digest)));
+
```

Comment:
Should we always update these metrics? I am not sure we should. Try to put all changes to them in the file only when there is a real change.
Also, can you please make sure again you update in all cases where next function are called in this file:
pruned_blocks_digests_.push_back(...)
pruned_blocks_digests_.erase(...)

Reply:
Thanks. I've moved code to place where pruning vector is updated.